### ISSUE TITLE
Implement IP rotation for musicbrainz scripts

### ISSUE THREAD

**Issue by TimHenkelmann** [MusicConnectionMachine/StructuredData#70]
While IP rotation can help us for all scrape scripts, this is especially needed for the musicbrainz scripts. GetArtists, GetWorks and GetReleases are getting executed in parallel and musicbrainz will return an error as soon as [more than one request per second](https://musicbrainz.org/doc/XML_Web_Service/Rate_Limiting) is sent.

**Comment by kordianbruck** [comment]
Is this really needed? I'm not sure how to implement this without proxys or something. If we can scrape the data once, its fine - we don't need to do it on a daily basis.

**Comment by kordianbruck** [comment]
Not needed for now.

**Comment by TimHenkelmann** [comment]
Okay, then the structure from the musicbrainz scrape execution needs to change. I will create a separate issue for that

**Comment by kordianbruck** [comment]
Why does the script need to change? Its working already - no?

**Comment by TimHenkelmann** [comment]
It works for ~80-90% of the data, see #81
